src/AccessibilityInsights.Core/Results/RuleResult.cs

```diff
@@ -55,6 +51,16 @@ public class RuleResult
         /// </summary>
         public HelpUrl HelpUrl { get; set; }
 
+        /// <summary>
+        /// Returns the display text for the issue.
+        /// </summary>
+        public string IssueDisplayText { get; set; }
+
+        /// <summary>
+        /// The link to fetch / display the issue
+        /// </summary>
+        public Uri IssueLink { get; set; }
```

Comment:
I feel like this should be something in a separate class and the extension should be the one responsible for its handling how it gets evaluated

Reply:
The extension does get a chance to evaluate it since it comes directly from the extension in the form of an object that contains issue display text and issue Link. Would it be clearer if we added some comments towards that effect in the IssueInformation object?

Reply:
same